### PR TITLE
Threshold detection figures

### PR DESCRIPTION
**Pull request by jzwart**
Adding a couple of figures based on the scenario where there is a nitrate flux threshold and we are evaluating our model's ability to correctly forecast the exceedance of this threshold. I arbitrarily chose thresholds that were ~75-90th percentile of nitrate flux for each site, but these can be altered if we find a threshold that is more meaningful. 

I'm not sure if I get a whole lot out of the fraction of forecasts exceeding threshold fig but it is helpful to get an idea of where the threshold is. 
image

I find the forecasting skill figure useful and a good summary fig: 
image

see #59 and #58 for more details on decisions.

### CODE REVIEW

File: 9_report/src/fig_error_v_leadtime.R [hunk 1]
```diff
@@ -86,6 +93,9 @@ fig_error_v_leadtime <- function(fig_ind, config_fig_yml, preds_ind, agg_nwis_in
                      xlab('Lead Time (days)') +
                      theme(legend.position = 'none',
                            axis.title.y = element_blank(),
+                           axis.text = element_text(size = 15),
+                           strip.text = element_text(size = 15),
+                           axis.title = element_text(size = 15),
```
Comment (aappling-usgs) [inline 9_report/src/fig_error_v_leadtime.R]: i'm not worried about this for now, but i wonder if we can create a shared theme at some point that takes out some need for repeating this text in many places. let's just keep an eye out and decide later whether that'd be useful.

File: 9_report/src/fig_preds_v_obs.R [hunk 2]
```diff
@@ -39,28 +44,33 @@ fig_preds_v_obs <- function(fig_ind, config_fig_yml, loadest_preds_ind, wrtds_pr
 
   # create the plot
   g <- ggplot(preds_obs, aes(x = obs_flux/1000, y = pred_flux/1000, color = flux_model)) +
-    geom_point() +
-    geom_blank(data = dummy, aes(x= x_val/1000, y = y_val/1000)) +
+    geom_point(size = 2) +
+    geom_blank(data = dummy, aes(x= x_val/1000, y = y_val/1000)) + # to make 1:1 axes
```
Comment (aappling-usgs) [inline 9_report/src/fig_preds_v_obs.R]: good comment :+1:

File: 9_report/src/fig_error_v_leadtime.R (hunk 1 above)
Comment (jzwart) [reply 9_report/src/fig_error_v_leadtime.R]: yup we can add text style / font size / fig size to `config.fig.yml`

Comment (aappling-usgs) [reply 9_report/src/fig_error_v_leadtime.R]: that's probably plenty for this poster. but what i meant was something like
```r
iemss_theme <- theme(
  axis.text = element_text(size = 15),
  strip.text = element_text(size = 15),
  axis.title = element_text(size = 15)
)
```
which could be defined in its own shared file as an R object and could be tacked on as `g <-
 ggplot(...) + ... + iemss_theme` within each of the figure code files. this saves the repeated calls to `element_text`.

**Pull request merged**